src/components/dropdowns/dropdown/Dropdown.js

```diff
@@ -0,0 +1,70 @@
+import React, {useState} from "react";
+import styled from "styled-components"
+import { dropdownWrapperCSS, dropdownOuterCSS, dropdownInnerCSS, dropdownArrowCSS } from "./DropdownCSS";
+import { ChevronUp, ChevronDown } from "../../../assets/icons";
+import { colors } from "../../colors/index";
+
+const Dropdown = ({outerText, innerText, width}) => {
+	
+	const [open, setOpen] = useState(false);
+
+	const toggleDropdown = () => {
+		setOpen(!open);
+	};
+
+	const ToggleStyles = (p) => {
+		if(open) {
+			return `
+				{
+					background: ${colors.gradientMain};
+				}
```

Comment:
would add a new color in the colors.js file so the gradient is vertical
`gradientMain: 'linear-gradient(92.58deg, #0DEFE1 0%, #78FF96 100%)'`